test/generators/html/Recent-X.html

```diff
@@ -28,8 +28,7 @@ <h1>Module <code><span>Recent.X</span></code></h1>
     <div class="spec type anchored" id="type-t">
      <a href="#type-t" class="anchor"></a>
      <code><span><span class="keyword">type</span> t</span>
-      <span> = 
-       <span>int <a href="Recent-Z-Y-X.html#type-t">Z.Y.X.t</a></span>
+      <span> = <span>int <a href="Recent-Z-Y-X.html#type-t">L.X.t</a></span>
```

Comment:
Is this https://github.com/ocaml/odoc/issues/594 ?

Reply:
No, this is a consequence of https://github.com/ocaml/odoc/pull/853/commits/277d8d05500ef124e274f6057d43b7a106e89ee8

Reply:
Thanks.